pygame_gui/elements/ui_drop_down_menu.py

```diff
@@ -23,6 +23,7 @@ class UIExpandedDropDownState:
 
     :param drop_down_menu_ui: The UIDropDownElement this state belongs to.
     :param options_list: The list of options in this drop down.
+    :param options_dict: Optional dictionary of methods to call when an option is chosen.
```

Comment:
I feel like this parameter should have a different name to avoid confusion with `options_list` and to better represent what it is. Something like `per_option_callbacks` but perhaps snappier - if you can think of anything?